test/switch.test.js

```diff
@@ -74,7 +74,7 @@ it("ignores other elements", () => {
   const rendered = result.children[0].children;
 
   expect(rendered.length).toBe(1);
-  expect(rendered[0].type).toBe(Route);
+  expect(rendered[0].type).toBe('b');
```

Comment:
What about this? Looks like it changes the behaviour for non-Route elements/components. Should we just ignore this or keep the backward compatibility? 🤔

Reply:
I think the current behavior is in line with expectations, it should be the behavior that developers need to control, and I observe that react-router also maintains consistent behavior.

Reply:
Ok, can we replace the test case with:
```js

it("matches regular components as well", () => {
  const Dummy = props => props.children;

  const result = testRouteRender(
    "/",
    <Switch>
      <Dummy path="/">Component</Dummy>
      <b>Bold</b>
    </Switch>
  );

  const rendered = result.children[0].children;

  expect(rendered.length).toBe(1);
  expect(rendered[0].type).toBe(Dummy);
});
```